lib/dynflow/flows/abstract.rb

```diff
@@ -32,6 +32,18 @@ def all_step_ids
       def flatten!
         raise NotImplementedError
       end
+
+      def self.new_from_hash(hash)
```

Comment:
are we able to rename this method as `load` or are we bound to this name by call from another structures?

Reply:
Switched to `encode` and `decode`